README.md

```diff
@@ -99,6 +100,22 @@ ZINBA was the least performant by a massive margin, requiring much more time to
 
 ## RNA-seq
 
+### Alignment & Quantification Methods
+
+**Title:** [Alignment and mapping methodology influence transcript abundance estimation](https://www.biorxiv.org/content/10.1101/657874v2)
+
+**Authors:** Avi Srivastava\* &  Laraib Malik \* et al.
+
+**Journal Info:** bioRXiv, October 2019
+
+**Description:** This paper compares the influence of mapping and alignment on the accuracy of transcript quantification in both simulated and experimental data, as well as the effect on subsequent differential expression analysis.
+
+**Tools/methods compared:** `bowtie2`, `STAR`, `quasi-mapping`, `Selective Alignment`, `RSEM`, `Salmon`.
+
+**Recommendation(s):** When trying to choose an approach, a choice can be made by the user performing the analysis based on any time-accuracy tradeoff they wish to make. In terms of speed, quasi-mapping is the fastest approach, followed by SA and SAF and then STAR. Bowtie2 was considerably slower than all three of these approaches. However, in terms of accuracy, SAF yielded the best results, followed by alignment to the genome (with subsequent transcriptomic projection) using STAR and SA (using carefully selected decoy sequences). Bowtie2 generally performed similarly to SA, but without the benefit of decoy sequences, seemed to admit more spurious mappings. Finally, lightweight mapping of sequencing reads to the transcriptome showed the lowest overall consistency with quantifications derived from the oracle alignments.
```

Comment:
Can you link to bowtie2/STAR/Salmon at least once each here? And note what SAF stands for? And maybe indicate that both quasi-mapping and selective alignment were done by Salmon?

Generally okay with this, as it's a true comparison of methods rather than "This is why Salmon is the greatest thing since sliced bread".

Reply:
Just pushed the requested changes. Thanks !